packages/manager/modules/pci/src/projects/project/storages/databases/database/query-statistics/query-statistics.controller.js

```diff
@@ -36,12 +37,15 @@ export default class {
       this.database.id,
     )
       .then(() => {
-        return this.goBackToQueryStatistics(
-          this.$translate.instant(
-            'pci_databases_query_statistics_reset_success',
-          ),
-          'success',
-        );
+        // added timeout as a work-around to avoid the aiven bug
```

Comment:
**question:** Which bug are you referring to in that particular case?

Reply:
@antleblanc added the bug number to the comment

Reply:
@antleblanc the bug related to aiven is PUD-1148